tests/cpu/transforms/BuiltinTransform_tests.cpp

```diff
@@ -365,128 +363,190 @@ AllValues UnitTestValues
 {
     // Contains the name, the input values and the expected output values.
 
-    { "IDENTITY",
-        { { 0.5f, 0.4f, 0.3f }, { 0.5f,            0.4f,            0.3f } } },
+    { "IDENTITY", 
```

Comment:
adding ability to specify different thresholds for different built-ins as some are more sensitive than others to the underlying implementation change (i.e. lut-based vs scalar vs SSE implementation). Increased the threshold for PQ curve which has nested power() invocation thus numerically less stable. Also adding extended range values to make sure that all implementations can handle wider than [0,1] domain. Target values are computed with double-precision math.